### PR TITLE
[verisure] Add channel for low battery warning for supported thing types

### LINKED ISSUE #10406: [verisure] Add channel for low battery warning for supported thing types
After some investigation I’ve found out that Verisure only supports 2 levels of battery supervision.

* “status”:“CRITICAL” - that indicates a low battery situation
* “status”:“OTHER” - that means that you have more than 6 months of battery life left

It is only supported for thing types:

* smokeDetector
* siren
* nightControl
* doorWindowSensor

I will implement a channel lowBattery that can be of Switch type that would then be on if battery level is CRITICAL for those thing types.

### PR DESCRIPTION
**Pull request by jannegpriv**
[verisure] Add channel for low battery warning for supported thing types (#10406)

Signed-off-by: Jan Gustafsson <jannegpriv@gmail.com>

Fixes #10406.

### CODE REVIEW

**Review by cweitkamp — CHANGES_REQUESTED** [review]
Thanks for your contribution.

File: bundles/org.openhab.binding.verisure/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -160,6 +160,7 @@
 			<channel id="installationName" typeId="installationName"/>
 			<channel id="installationId" typeId="installationId"/>
 			<channel id="smokeDetectorTriggerChannel" typeId="triggerChannel"/>
+			<channel id="lowBattery" typeId="lowBattery"/>
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.verisure/src/main/resources/OH-INF/thing/thing-types.xml:163]: There is no need to introduce a new channel type. We have a [system default channel type](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) for `low-battery`

```suggestion
			<channel id="lowBattery" typeId="system.low-battery"/>
```

Comment (jannegpriv) [reply bundles/org.openhab.binding.verisure/src/main/resources/OH-INF/thing/thing-types.xml]: Fixed!

**Review by cweitkamp — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**